packages/radix-ui-themes/src/components/data-list.props.ts

```diff
@@ -0,0 +1,79 @@
+import { PropDef, trimProp, widthPropDefs } from '../helpers';
+
+const orientationValues = ['horizontal', 'vertical'] as const;
+const gapValues = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'] as const;
+const alignValues = ['start', 'center', 'end', 'baseline'] as const;
+const sizes = ['1', '2', '3'] as const;
```

Comment:
We need to decide visually how we want these three sizes to look. Right now they map to the corresponding `Text` sizes.

Reply:
<img width="560" alt="Screenshot 2024-01-17 at 5 34 14 PM" src="https://github.com/radix-ui/themes/assets/36613477/fd59a86b-52d4-4ef2-868b-6ff88aeb8785">

Reply:
I'd say we support all text sizes because if you put a CSS class on it manually it's going to work anyway

Size 9 is a bit stupid, but so are most size 9 things 🙂